### ISSUE TITLE
Please upload the mod to modrinth, including the 1.12.2 version

### ISSUE THREAD

**Issue by mohammedthejihadist** [PrinceOfAmber/Storage-Network#523]
Modrinth is the superior modding platform, both from an ethical and functional point of view. In addition, everyone is instantly eligible for monetization.

**Comment by Lothrazar** [comment]
yes i will . it will exist here once its ready https://modrinth.com/mod/simple-storage-network
